packages/gestalt/src/ActivationCard.js

```diff
@@ -36,7 +36,7 @@ type Props = {|
    * Accessibility: `accessibilityLabel` populates aria-label.
    */
   dismissButton?: {|
-    accessibilityLabel: string,
+    accessibilityLabel?: string,
```

Comment:
Just for the record, could you explain why we're making this optional and why we make it required for `CompletedCard`?

Reply:
Some of this labels could be deprecated. Because DefaultLabelProvider covers them, they shouldn't be required.

Reply:
CompletedCard is a subcomponent and should always have it from the API or the DefaultLabelProvider.